ui/components/component-library/input/input.tsx

```diff
@@ -47,7 +47,7 @@ export const Input: InputComponent = React.forwardRef(
         },
         className,
       )}
-      aria-invalid={error}
+      {...(error && <>aria-invalid={error}</>)} // This stops it from including the attribute when `aria-invalid="false"`
```

Comment:
Interesting use of fragments here. For conditionally rendering props we use this convention I wonder if it would be more consistent this way? 
```suggestion
      {...(error ? { 'aria-invalid': error } : {})}
```